api/management/api.go

```diff
@@ -117,6 +117,8 @@ type RealmCustomConfiguration struct {
 	ShowPasswordTab                     *bool   `json:"show_password_tab"`
 	ShowMailEditing                     *bool   `json:"show_mail_editing"`
 	ShowAccountDeletionButton           *bool   `json:"show_account_deletion_button"`
+	CancelRegistrationURL               *string `json:"cancel_registration_url,omitempty"`
+	ConfirmedRegistrationURL            *string `json:"confirmed_registration_url,omitempty"`
```

Comment:
I would suggest ConfirmRegistrationURL so that it respects the other names

Reply:
New name suggestion is:
- RedirectSuccessfulRegistrationURL
- RedirectCancelledRegistrationURL

Reply:
Done